static/app/views/settings/components/forms/controls/rangeSlider.tsx

```diff
@@ -1,13 +1,23 @@
-import * as React from 'react';
+import React, {ChangeEvent, KeyboardEvent, MouseEvent, useEffect, useState} from 'react';
 import styled from '@emotion/styled';
 
 import {t} from 'app/locale';
 import space from 'app/styles/space';
+import {defined} from 'app/utils';
 import Input from 'app/views/settings/components/forms/controls/input';
 
 type Props = {
   name: string;
 
+  /**
+   * String is a valid type here only for empty string
+   * Otherwise react complains:
+   * "`value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components."
```

Comment:
Nit, long comments like this are difficult to read in 80 character editor panes. I would suggest wrapping these type of things (using something like https://marketplace.visualstudio.com/items?itemName=stkb.rewrap#:~:text=The%20main%20Rewrap%20command%20is,multiple%20comments%20in%20one%20selection. maybe)

Reply:
Oh I see this just moved from somewhere else haha

Reply:
sure thing, but this code was just moved. I think I would prefer to add more improvements in a follow-up PR if that is fine